servers/shared-snowplow-consumer/src/eventConsumer/collectionEvents/types.ts

```diff
@@ -1,31 +1,10 @@
-export type BasicCollectionEventPayloadWithContext = {
-  object_version: string;
-  collection: Collection;
+export type CollectionEventBridgePayload = {
```

Comment:
These are the types we could move to a package

Reply:
by this you mean the auto-gen snowtypes?

Reply:
It does, but it generated some interesting looking types